boards/arm/tmo_dev_edge/tmo_dev_edge.dts

```diff
@@ -101,6 +103,19 @@
 		regulator-name = "bq24250rge";
 		regulator-boot-on;
 	};
+
+	adc0:adc0@40002000 {
+		compatible = "sbs,sbs-battery";
+        status = "okay";
+		reg = <0x40002000 0x400>;
```

Comment:
The name should not be SBS battery, also this should be a general ADC not specific to our particular configuration